App Localisation

Mahm0ud98 opened the pull request:
Supporting both English and Arabic languages for the app
Fixing the dropdown list so its now show what is selected

Mahm0ud98 committed:
localization

asgatech20 commented on lib/bloc_prov/bloc_prov.dart:
```diff
@@ -0,0 +1,34 @@
+        builder: (context, local) {
+          context.read<LanguageCubit>().changeStartLang();
+          return MaterialApp(
+            // onGenerateTitle: (context) {
```
remove unnecessary comment

asgatech20 commented on lib/cubit/language_cubit.dart:
```diff
@@ -0,0 +1,22 @@
+class LanguageCubit extends Cubit<Locale> {
+  LanguageCubit() : super(const Locale('en'));
+
+  void changeStartLang() async {
```
change the name of function

asgatech20 commented on lib/cubit/language_cubit.dart:
```diff
@@ -0,0 +1,22 @@
+    }
+  }
+
+  void changeLang(context, String data) async {
```
it is the same function changeStartLang please remove one of them

Mahm0ud98 committed:
1st comment is done.
changed name of the function is done.
merging 2 functions cannont be done as each function of the 2 function do different job and cannot be initialized into only 1 function.
finished building the home screen with cubit

Mahm0ud98 committed:
Localization if 2nd screen
Remaining Icons for 2nd screen
